docs/install/_linux.rst

```diff
@@ -136,6 +109,35 @@ To change the system service owner, you need to change the configuration of the
 .. note:: Replace the ``$USER`` in commands above with the name of the user you wish to be the new Kolibri system service owner.
 
 
+Optimized Performance with ``kolibri-server`` package
```

Comment:
"Optimized" isn't quite the right word here.

Maybe "Higher Performance with the `kolibri-server` package"

Reply:
Done!